### DISCUSSION TITLE
Wonder how to create a plugin for AppFlowy Editor?

### DISCUSSION THREAD

**Discussion by annieappflowy** [AppFlowy-IO/AppFlowy#1124] [General]
https://github.com/AppFlowy-IO/appflowy-editor/blob/main/documentation/customizing.md

**Comment by aleripe** [comment]
Hi Annie, are all plugins at the block level? Is it actually possible to create an inline widget (like for example showing a little image inside text)?

Thanks!

**Reply by annieappflowy** [reply]
You've raised an excellent point. Can you provide us with a few use cases for inline widgets? For example, under what situations you would need it and how you will benefit from it.

Thanks!

**Comment by Adda0** [comment]
> https://github.com/AppFlowy-IO/AppFlowy/blob/main/frontend/app_flowy/packages/appflowy_editor/documentation/customizing.md

This link is outdated and does not work any more. I could not find the new path where the file (which was apparently renamed) is located due to the project being restructured. Could you update the link or point to up-to-date instructions for this topic?

What I found is https://appflowy.gitbook.io/docs/essential-documentation/appflowy-plugins. Was there anything specific in the original `customizing.md` file that this found documentation is missing?

**Reply by annieappflowy** [reply]
https://github.com/AppFlowy-IO/AppFlowy-Docs/pull/82

**Reply by annieappflowy** [reply]
Confirmed not working

**Reply by annieappflowy** [reply]
Please refer to https://github.com/AppFlowy-IO/appflowy-editor/blob/main/documentation/customizing.md

**Reply by Adda0** [reply]
Thank you kindly. Have a great day.

For reference: The link I posted points to different documentation. Use the main link to `customizing.md`.
